packages-next/testing/src/index.ts

```diff
@@ -0,0 +1,91 @@
+import path from 'path';
+import crypto from 'crypto';
+import fs from 'fs';
+import supertest, { Test } from 'supertest';
+import memoizeOne from 'memoize-one';
+import { initConfig, createSystem, createExpressServer } from '@keystone-next/keystone';
+import { pushPrismaSchemaToDatabase } from '@keystone-next/keystone/migrations';
+import {
+  getCommittedArtifacts,
+  writeCommittedArtifacts,
+  requirePrismaClient,
+  generateNodeModulesArtifacts,
+} from '@keystone-next/keystone/artifacts';
+import type { KeystoneConfig, KeystoneContext } from '@keystone-next/types';
+
+export type GraphQLRequest = (arg: {
+  query: string;
+  variables?: Record<string, any>;
+  operationName?: string;
+}) => Test;
+
+export type TestArgs = {
+  context: KeystoneContext;
+  config: KeystoneConfig;
```

Comment:
Is this used and if so, why is this better than assigning the config to a variable and just using that?

Reply:
This is used to make the final config object used by the test runner available to the test, since we make changes to the value that's passed in. This was more useful when we were doing more manipulations to the config (e.g. iterating over providers, database schema, etc), but I think it's still helpful as is for cases where the original config object might not be easily available in the place where the test function is run.

That said I don't think we explicitly depend on it anywhere any more, so we could remove it if needed. I'm only marginally in favour of keeping it.

Reply:
Could we remove it then? And if we need it in the future, we can add it back?